apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -124,6 +124,23 @@
 			@close="onMenuClose">
 			<template v-if="share">
 				<template v-if="share.canEdit">
+					<!-- Custom Label -->
+					<ActionInput
+						ref="label"
+						v-tooltip.auto="{
+							content: errors.label,
+							show: errors.label,
+							trigger: 'manual',
+							defaultContainer: '#app-sidebar'
```

Comment:
```suggestion
							defaultContainer: '#app-sidebar'
```

Not valid for `@nextcloud/vue` >= 2.0.0 https://github.com/nextcloud/nextcloud-vue/releases/tag/v2.0.0